rotkehlchen/db/dbhandler.py

```diff
@@ -1631,6 +1631,7 @@ def delete_data_for_ethereum_address(self, address: ChecksumEthAddress) -> None:
         cursor.execute(f'DELETE FROM used_query_ranges WHERE name="aave_events_{address}";')
         cursor.execute('DELETE FROM ethereum_accounts_details WHERE account = ?', (address,))
         cursor.execute('DELETE FROM aave_events WHERE address = ?', (address,))
+        cursor.execute('DELETE FROM multisettings WHERE value = ?', (address,))
```

Comment:
you also need tospecify the name of the setting here. This is too generic.

A `LIKE queried_address_%` comparison for the name. Check how it's done here: https://github.com/rotki/rotki/blob/313c2b57fba078b7a20f138a188a73a8696cabf1/rotkehlchen/db/dbhandler.py#L696

And yes a test would be nice. Maybe somewhere here? https://github.com/rotki/rotki/blob/313c2b57fba078b7a20f138a188a73a8696cabf1/rotkehlchen/tests/api/test_blockchain.py#L1538

Reply:
Ok so you also have more stuff planned for there I will do.